webapi/webapi-webgl-khronos-tests/webgl/khronos/COPYING

```diff
@@ -93,6 +93,63 @@ function testPassed(assertName, name) {
  }
 
 
+Remove unused WebGL test cases:
```

Comment:
@zqzhang , These removed tests are not included in [webgl-conformance-tests](https://www.khronos.org/registry/webgl/conformance-suites/1.0.3/webgl-conformance-tests.html). Some of them may cause browser crash or run a long time, the others are performance tests. So, the Khronos conformance test suite doesn't include these tests.